apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/style.scss

```diff
@@ -0,0 +1,7 @@
+.edit-post-fullscreen-mode-close__toolbar {
```

Comment:
This class should only be active for the wp_template_part and page post types. Currently this will hide the back button in the post editor as well, which we don't want to touch :) 

<img width="1511" alt="Screen Shot 2019-10-31 at 12 13 12 PM" src="https://user-images.githubusercontent.com/6265975/67978643-d5803380-fbd7-11e9-83ac-ff964937eb00.png">

Reply:
I think there may be a `.full-site-editing` class added to the body element for pages and wp_template_part, but I'm not sure if that class also exists on the post editor. if it doesn't, then you could use it to narrow down the scope a bit :)

Reply:
While the `.full-site-editing` class is not added here, I have updated `styles.scss` to use the corresponding `post-type-$typeName` classes so that this does not hide the button for the posts editor.

Reply:
this change looks great!